topics/application-gradle-plugin.md

```diff
@@ -0,0 +1,68 @@
+[//]: # (title: Distribution via Gradle Application plugin)
+
+From the Gradle [documentation](https://docs.gradle.org/current/userguide/application_plugin.html):
+> The Application plugin facilitates creating an executable JVM application. 
+> It makes it easy to start the application locally during development, 
+> and to package the application as a TAR and/or ZIP including operating system specific start scripts.
+
+We will use the second responsibility of this plugin and show how to package a simple Ktor application.
+
+Let's start with the following project structure:
+
+```text
+├── build.gradle
+├── settings.gradle
+└── src
+    └── main
+        ├── kotlin
+        └── resources
+├── gradle
+├── gradlew
+└── gradlew.bat
+```
+You can use the [wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:adding_wrapper) task to generate `gradle`, `gradlew` and `gradlew.bat` files.
```

Comment:
We <> You. The paragraph above using We, in this one You. We need to keep it consistent. @andreyaksenov have we settled on We or You?

Reply:
I use `You`